pkg/scheduler/cache/cache.go

```diff
@@ -1042,19 +1081,31 @@ func (sc *SchedulerCache) processResyncTask() {
 		return
 	}
 
+	klog.V(5).Infof("the length of errTasks is %d", sc.errTasks.Len())
+
 	defer sc.errTasks.Done(obj)
 
-	task, ok := obj.(*schedulingapi.TaskInfo)
+	taskKey, ok := obj.(string)
 	if !ok {
-		klog.Errorf("failed to convert %v to *schedulingapi.TaskInfo", obj)
+		klog.Errorf("Failed to convert %v to string.", obj)
```

Comment:
The obj format is incorrect. Is it more reasonable to exclude Forget?

Reply:
done